### DISCUSSION TITLE
[gatsby-theme-minimal-blog]: Support customizing the bottom area of a Post

### DISCUSSION THREAD

**Discussion by mactavishz** [LekoArts/gatsby-themes#698] [Ideas / Feature Requests] [answered]
Hi, I was trying to add a comment section to my blog. I found out that in order to achieve this, I must shadow the whole `post.tsx` by simply copy-pasting the source code. Because `post` component isn't design to have any child components.

```jsx
<Layout>
    {/* ... */}
    <section
      sx={{
        my: 5,
        ".gatsby-resp-image-wrapper": { my: [4, 4, 5], boxShadow: shadow.join(`, `) },
        variant: `layout.content`,
      }}
    >
      <MDXRenderer>{post.body}</MDXRenderer>
    </section>
    {/* here is where I could put my comment component  */}
    <DiscussionEmbed { ...{ ...disqusConfig, config: { identifier: post.slug, title: post.title } } } />
 </Layout>
```

My idea: This could simply be done by providing an empty component such as `post-bottom` that can be shadowed by the user. Or simply allow this component to have child components.

**Comment by Aryaman73** [comment]
Not an answer: but for adding comments, did you fork off of `gatsby-theme-minimal-blog` or `gatsby-theme-minimal-blog-core`?

**Reply by mactavishz** [reply]
No, I didn't fork them. I just shadowed the entire `Post` component and inserted my own component in it.

**Comment by LekoArts** [comment]
Hi!

Thanks for the suggestion 👍 This totally makes sense and I think an additional component that is empty by default and that one can shadow is the solution what I'd prefer.

**Comment by LekoArts** [comment, marked as answer]
https://github.com/LekoArts/gatsby-themes/pull/705 was published :)

**Reply by mactavishz** [reply]
Thanks a lot 👍 🥳
